Sprint-1/fix/median.js

```diff
@@ -3,9 +3,15 @@
 // If you're in the Sprint-1 directory, you can run `npm test -- fix` to run the tests in the fix directory
 
 function calculateMedian(list) {
+  let isEven = list.length % 2 == 0;
```

Comment:
1) The arrays specified in `median.test.js` happened to be sorted.
Normally, before we can easily find the median value in an array, we need to sort the array first.

2) Array is passed to a function ***by reference***  (instead of ***by value***). This implies when an array is passed to a function via parameter, the array is **shared** by the caller and the callee (the function being called).

To avoid modifying the caller's array, a common practice is to make a clone of the array (that's passed as a parameter) and modify the clone in the callee instead.

---
Can you try inserting code to first clone the array `list` and then sort the numbers in cloned array?
You may have to rename the array in lines 6-12.

Reply:
sure i can do it !!